### DISCUSSION TITLE
Example switching between top menus?

### DISCUSSION THREAD

**Discussion by dbuzz111** [tui-cs/Terminal.Gui#2304] [Q&A]
I've google to no end and can't seem to find anyone talking about the correct way to do top menu's with Termial.Gui can anyone point me at an example or explaner on the best to way to have multiple top menu items and switch between them?

Thank you!

**Comment by tznind** [comment]
Hi, Hope you are doing well and enjoying the library.  Do you mean having multiple menus that the user can switch between?  For example:

![image](https://user-images.githubusercontent.com/31306100/214219896-384ccc95-22d4-4638-92af-fa46d35384f6.png)

If so then there are several scenarios that have multiple top level menus for example [TableEditor](https://github.com/gui-cs/Terminal.Gui/blob/3d75be4ea60aa670544e39394b44adf6733a39df/UICatalog/Scenarios/TableEditor.cs#L50-L81) has both a `File` and `View` menu:

https://github.com/gui-cs/Terminal.Gui/blob/3d75be4ea60aa670544e39394b44adf6733a39df/UICatalog/Scenarios/TableEditor.cs#L50-L81

Another option if you find the above snippet too daunting would be using [the TerminalGuiDesigner dotnet tool](https://github.com/gui-cs/TerminalGuiDesigner/#terminal-gui-designer):

If you follow the [Hello World tutorial](https://github.com/gui-cs/TerminalGuiDesigner/blob/main/README.md#usage) but create a `TopLevel` instead of a `Window` you should get the below experience:

```
dotnet new console -n hello
cd hello
dotnet add package Terminal.Gui

dotnet tool install --global TerminalGuiDesigner
TerminalGuiDesigner -v Toplevel MyTopLevel.cs
```

![makemenus](https://user-images.githubusercontent.com/31306100/214224311-78542198-d24c-4d65-ada5-450840a972c0.gif)
_You can also use Ctrl+T to assign a menu shortcut and Ctrl+R to rename the class instance variable for the menu name_
